tests/test_load_data.py

```diff
@@ -29,6 +30,21 @@ def test_roundtrip_simple(test_data_dir, tmp_path):
     assert len(list(dsnpy.keys())) == 1
 
 
+def test_roundtrip_no_operation_all_field(test_data_dir, tmp_path):
+    copytree(test_data_dir / "idefix_spherical_planet3d", tmp_path / "mydir")
+
+    os.chdir(tmp_path / "mydir")
+    ds = GasDataSet(500)
+    assert len(list(ds.keys())) == 7
+
+    gf = ds["RHO"]
+
+    gf.save()
+    dsnpy = GasDataSet.from_npy(500, operation="")
+    assert len(list(dsnpy.keys())) == 1
+    np.testing.assert_array_almost_equal(ds["RHO"].data, dsnpy["RHO"].data)
```

Comment:
Maybe there's something I'm missing and this doesn't work, but I would naively assume roundtrips preserves data exactly ?
```suggestion
    np.testing.assert_array_equal(ds["RHO"].data, dsnpy["RHO"].data)
```